cl/recap/mergers.py

```diff
@@ -1734,8 +1766,21 @@ async def merge_attachment_page_data(
             try:
                 rd = await RECAPDocument.objects.aget(**params)
             except RECAPDocument.DoesNotExist:
-                rd = RECAPDocument(**params)
-                rds_created.append(rd)
+                try:
+                    doc_id_params = deepcopy(params)
+                    del doc_id_params["attachment_number"]
+                    del doc_id_params["document_type"]
+                    doc_id_params["pacer_doc_id"] = attachment["pacer_doc_id"]
+                    rd = await RECAPDocument.objects.aget(**doc_id_params)
+                    if attachment.get("attachment_number") == 0:
```

Comment:
Thanks, this now makes sense. Just to confirm a couple of facts:

- Attachment number 0 must always be the Main Document.
- The pacer_doc_id from the docket entry can sometimes mismatch the `pacer_doc_id` from the Main Document on the attachment list.

Correct?

Reply:
> * Attachment number 0 must always be the Main Document.

I think so, I mean at least I think the short description always matches attachment 0(if it exists) which is the top document in the attachment page.

> The pacer_doc_id from the docket entry can sometimes mismatch the `pacer_doc_id` from the Main Document on the attachment list.

Yeah, seems it may say sometimes match the last attachment.